andrada12/assignment3/WordGrid.java

```diff
@@ -0,0 +1,106 @@
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.Scanner;
+
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+/**
+ *
+ * @author andrada
+ */
+public class WordGrid {
+    int[] Ox = new int[]{-1,0,1};
+    int[] Oy = new int[]{-1,0,1};
+    
```

Comment:
These variable names aren't very explanatory, and especially as O and 0 are easily confused make your code a little hard to read. Can you think of a good name?
I'd use something like "xSteps" or "xNeighbours"